pages/index.tsx

```diff
@@ -76,6 +76,7 @@ const Home: NextPage<HomeProps> = ({ projects }) => {
 					</List>
 				</Container>
 			</Box>
+			<RecentProjectActivity projects={projects} />
```

Comment:
👏

Reply:
Also, the reason this one is abstracted out to it's own component as opposed to the rest is:

1. There's possibility that it can easily be put in elsewhere in the UI, i.e. projects page or project details pages, or a sidebar variant, etc.
2. Its cleaner.

Its cleaner if we move everything out from top-level `page/` components and keep them as clean as possible, with only layout and server side prop concerns.  So all these `Box` components should follow suit like `RecentProjectActivity` eventually.